lti/views.py

```diff
@@ -38,25 +40,30 @@ def dispatch(self, request, *args, **kwargs):
 
         # validate the request
         lti = LTI(request_type="any", role_type="any")
-        try:
-            lti.verify(request)
-        except LTIException:
-            return render(request, "lti_failure.html")
+        lti.verify(request)
+        # try:
+        #     lti.verify(request)
+        # except LTIException:
+        #     logger.debug("The error is in lti verigy")
+        #     return render(request, "lti_failure-verify.html")
 
         if not request.user.is_authenticated:
             try:
                 lti_user = login_existing_user(request)
             except EmailAddress.DoesNotExist:
                 lti_email = request.POST.get(LTI_PROPERTY_USER_EMAIL, None)
                 if not lti_email:
-                    return render(request, "lti_failure.html")
+                    logger.debug("The error is there is no lti_email")
+                    return render(request, "lti_failure-email.html")
```

Comment:
Yeah, totally. This was just something I added in quickly to debug the lti config issue we were seeing. This will also be reverted to the way it was, I believe, before the PR is supposed to be ready for review